data-prepper-plugins/otel-proto-common/build.gradle

```diff
@@ -7,6 +7,10 @@ plugins {
     id 'java'
 }
 
+test {
+    jvmArgs '-Xmx4g'
```

Comment:
This is rather concerning. Why do we need 4 GB for the tests? 

Also, we should avoid making changes like this in the sub projects. It is hard to control from the root level.

Reply:
It looks like 512m is enough. So, changed it to `jvmArgs '-Xmx512m'` Without this change the test wont' be able to create a LogsRequest size more than 8MB.